process/security_baseline.md

```diff
@@ -0,0 +1,254 @@
+
+As technology and the threat landscape evolve, the baseline will continuously adapt. The primary goal of this initiative is to maintain a balance between security, reliability, performance, and cost-effectiveness.
+
+The key words “MUST”, “MUST NOT”, “REQUIRED”, “SHALL”, “SHALL NOT”, “SHOULD”, “SHOULD NOT”, “RECOMMENDED”, “MAY”, and “OPTIONAL” in this document are to be interpreted as described in [RFC 2119](https://www.rfc-editor.org/rfc/rfc2119/).
+
+## Background
+The initiative was one of the outcomes of the October 2023 Linux Foundation Member Summit. Making open source software more secure is one of the top priorities across the Linux Foundation. OpenSSF is leading the charge. 
+
+The security of open source software is a matter of global interest and concern.  In the United States, open source software is used across all critical infrastructure sectors defined by CISA (Cybersecurity and Infrastructure Security Agency), for example, health care, defense, financial services, utilities, telecommunications, etc. Open source security directly impacts national security, economics and social stability. Enhancing open source security is imperative. NIST has published a [Secure Software Development Framework](https://csrc.nist.gov/Projects/ssdf) (SSDF) as a result of Executive Order (EO) 14028 on "[Improving the Nation's Cybersecurity](https://www.federalregister.gov/documents/2021/05/17/2021-10460/improving-the-nations-cybersecurity)".
+
+In the European Union, [Cyber Resiliency Act](https://eur-lex.europa.eu/legal-content/EN/TXT/PDF/?uri=CONSIL:ST_17000_2023_INIT) (CRA) has declared a new role - Open Source Steward(Article 3(18a). The legislation is to ensure that consumers of products with digital features are protected, and manufacturers of these products are held accountable for product security. Open source security is the foundation for manufacturers to be compliant with the legislation.     
+
+## Success Criteria
+The success of the security baseline SHOULD be quantified and qualified in a few dimensions:
+ * **Adoption and Advancement of Security Technologies**
+   * **Objective**: The Security Baseline promotes improved open source security outcomes.
+   * **Approach**: Utilize technical outcomes from the OSS community to implement, and verify the security baseline.
+   * **Collaborations**: Partner with other foundations to enhance OSS security through education, specifications, frameworks, and tooling.
+   * **Automation and Tooling**: Identify opportunities for automation and tooling to ease the burden on maintainers and contributors, facilitating secure by design and secure by default principles.
+ * **Increased Security Baseline Adoption Rate**
+   * **Objective**: Ensure the Security Baseline is widely adopted.
+   * **Metrics**:
+     * Three OpenSSF software-based pilot projects meet the baseline requirements for each project's life cycle by 9/15/2024.
```

Comment:
This sounds reasonable, as mentioned on https://github.com/ossf/tac/issues/215 both @protobom and @openvex will start piloting the baseline as soon as it is merged. Especially protobom is still working on setting its new org, so its an ideal time for us.